docs/user_guide/assessment.rst

```diff
@@ -407,12 +407,55 @@ to take advantage of the inherent plotting capabilities of
 
 .. literalinclude:: ../auto_examples/plot_quickstart.py
     :language: python
-    :start-after: # Analyze metrics using MetricFrame
+    :lines: 32-48
 
 .. figure:: ../auto_examples/images/sphx_glr_plot_quickstart_001.png
     :target: auto_examples/plot_quickstart.html
     :align: center 
 
+It is possible to customize the plots. Here are some common examples:
+
+Customize Plots: `ylim`
+'''''''''''''''''''''''
```

Comment:
Does this way of underlining work? I think `~` works for sure. Our doc build preview doesn't currently work so I haven't checked yet, but please briefly check and let us know if this way works, too.

Reply:
I still couldn't get my local to build with the theme but I referred to the [sphinx docs](https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings). Here is my local build without the theme, I meant for the customisation headings to be H6, under the `Plotting Grouped Metrics` section (H5)

<img width="968" alt="image" src="https://user-images.githubusercontent.com/52529078/119143326-11107600-ba7a-11eb-8adb-f3bc0aec6aa3.png">
<img width="374" alt="image" src="https://user-images.githubusercontent.com/52529078/119141606-0bb22c00-ba78-11eb-82b1-8e570e628316.png">

Reply:
It should be fixed now. There was a doc build issue that's now resolved as of today #815

Reply:
To elaborate: you may have to pull in the latest `main` branch. It might automatically just merge it, so I've kicked off the doc build again, but for yourself locally you'll need to pull it in.

Reply:
Can you make sure the `'''''''` render like `~~~~~~~` when using in a header? Or better, just replace it for uniformity with other headers?